x/validate/module.go

```diff
@@ -57,8 +57,10 @@ func (a AppModule) TxValidator(ctx context.Context, tx transaction.Tx) error {
 		}
 	}
 
-	if err := a.feeTxValidator.ValidateTx(ctx, tx); err != nil {
-		return err
+	if a.feeTxValidator != nil {
```

Comment:
we should panic if its not set here as this could be a footgun for users where a chain would start and the app developer may have forgotten to set this up correctly. Maybe we add a `if == nil {panic} call here

Reply:
+1 on this